pulpcore/app/models/content.py

```diff
@@ -498,23 +498,24 @@ def natural_key_fields(cls):
         """
         return tuple(chain.from_iterable(cls._meta.unique_together))
 
+    @classmethod
+    def _sanitized_natural_key_fields(cls):
```

Comment:
`from functools import lru_cache`

```suggestion
    @lru_cache(typed=True)
    def _sanitized_natural_key_fields(cls):
```

Should speed this up.

Reply:
Can you leave a docstring for this method to indicate why it's needed?